File: 1475.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int a;
int arr[10];

int main(){
    cin >> a;

    while(a>0){
        arr[a%10]++;
        a /= 10;
    }

    int ans = 0;

    arr[6] += arr[9];
    arr[6] = arr[6]/2 + arr[6]%2;
    arr[9] = arr[6];

    for(int i=0; i<10; i++){
        ans = max(ans, arr[i]);
    }

    cout << ans;
    return 0;
}

File: 1475.py
import sys

n = int(sys.stdin.readline())

arr = [0] * 10

while(n>0):
    arr[n%10] += 1
    n //= 10

temp = arr[6] + arr[9]
arr[6] = 0
arr[9] =  temp//2 if temp%2 == 0 else temp//2 + 1

print(max(arr))